wdqs-frontend/latest/default.conf

```diff
@@ -14,7 +14,7 @@ server {
     }
 
     location / {
-        root   /usr/share/nginx/html;
+        root   /usr/share/nginx/html/build;
```

Comment:
I would not put it here. I will change the line 37 from:
```COPY --from=builder /tmp/wikidata-query-gui-master /usr/share/nginx/html```
To
```COPY --from=builder /tmp/wikidata-query-gui-master/build /usr/share/nginx/html````
That way we only copy the build to the docker image.

Reply:
yes, amended commit: 83d44b2